services/github/src/handlers/utils.js

```diff
@@ -232,7 +232,7 @@ export const getTimeDifference = (timestamp1, timestamp2) => {
 };
 
 export const buildUrl = (rootUrl, taskId, runId, artifactName) => {
-  return `${rootUrl}/tasks/${taskId}/runs/${runId}/${artifactName}`;
+  return `${rootUrl}/tasks/${taskId}/runs/${runId}/logs/${artifactName}`;
```

Comment:
`/logs` was probably a special case of artifacts, sorry. We'd likely need to check the name also: https://github.com/taskcluster/taskcluster/blob/main/ui/src/views/Tasks/routes.js#L63-L71

And maybe this only works for the logs.. 
I think we can only link those two logs (log, live log) directly, and all other artifact links should just point to the task page (so url ends with runId) 
Thanks!